influxdata/influxdb
# refactor(ui): replace local Tabs with Clockface Tabs

Closes #14696

- Replace all instances of locally defined `Tabs` with the Clockface version
- Create a shared component for the consistent rendering of settings pages
- Remove unused logic from when the tasks page was displayed in 2 places
- Add `testID` prop to `SearchWidget` component
- Ensure all settings pages use `SearchWidget` for consistency's sake
- Delete locally defined `Tabs` components

![Screen Shot 2019-08-16 at 1 06 02 PM](https://user-images.githubusercontent.com/2433762/63195126-a1f52a80-c026-11e9-865e-66da1176ee02.png)



- [ ] http/swagger.yml updated (if modified Go structs or API)
